styles/Search.module.css

```diff
@@ -0,0 +1,25 @@
+.textfield {

```

Comment:
Since Epic React teaches module.css instead of styled components, we're letting this go. But, SP normally uses Styled Components or TailwindCSS Styled components.